application/CMakeLists.txt

```diff
@@ -170,12 +170,8 @@ endif()
 
 # Windows executable without console
 if(WIN32)
-  if(F3D_WINDOWS_GUI)
-    set_target_properties(f3d PROPERTIES WIN32_EXECUTABLE TRUE)
-  else()
-    # force usage of wWinMainCRTStartup in console mode for Unicode support
-    list(APPEND f3d_link_options_public "/ENTRY:wWinMainCRTStartup")
-  endif()
+  # force usage of wWinMainCRTStartup in console mode for Unicode support
+  list(APPEND f3d_link_options_public "/ENTRY:wWinMainCRTStartup")
```

Comment:
that means F3D run with a console open always, not sure we want that

Reply:
Considering the code we had before I don't think so. When `F3D_WINDOWS_GUI` was OFF we had no console.  
The documentation is here: https://learn.microsoft.com/en-us/cpp/build/reference/entry-entry-point-symbol?view=msvc-170